generic/src/test/java/org/datahem/processor/generic/GenericStreamPipelineTest.java

```diff
@@ -0,0 +1,600 @@
+	
+	private static final Logger LOG = LoggerFactory.getLogger(GenericStreamPipelineTest.class);
+	
+	@Rule public transient TestPipeline p = TestPipeline.create();
+
+    
+	private Map<String,String> attributes = new HashMap<String, String>(){
+		{
+            put("timestamp", "2013-08-16T23:36:32.444Z");
+            put("uuid", "123-456-abc");
+            put("source", "test");
+		}
+	};
+
+
+	@Test
+	public void withoutOptionsTest(){
+
+        JSONObject testPayloadObject = new JSONObject()
+            .put("StringField", "a string")
+            .put("Int32Field", 32) 
+            .put("Int64Field", 64) 
+            .put("DoubleField", 1.1) 
+            .put("FloatField", 1) 
+            .put("BoolField", true) 
+            .put("BytesField",  Base64.getEncoder().encodeToString("bytes".getBytes())) 
+            .put("EnumField", 1)
+            .put("repeatedString", new JSONArray().put("one").put("two").put("three"))
+            .put("repeatedInt32", new JSONArray().put(32).put(64).put(128))
+            .put("repeatedInt64", new JSONArray().put(64).put(128).put(256))
+            .put("repeatedDouble", new JSONArray().put(1.1).put(1.2).put(1.3))
+            .put("repeatedFloat", new JSONArray().put(1).put(2).put(3))
+            .put("repeatedBool", new JSONArray().put("true").put("false").put("true"))
+            .put("repeatedBytes", new JSONArray()
+                .put(Base64.getEncoder().encodeToString("bytes".getBytes()))
+                .put(Base64.getEncoder().encodeToString("bytes".getBytes()))
+                .put(Base64.getEncoder().encodeToString("bytes".getBytes())))
+            .put("repeatedEnum", new JSONArray().put(0).put(1).put(2));
+        
+        JSONObject messageChild = new JSONObject(testPayloadObject.toString());
+        messageChild.remove("_ATTRIBUTES");
+        messageChild.put("stringMap", new JSONObject()
+            .put("timestamp", "2013-08-16T23:36:32.444Z")
+            .put("uuid", "123-456-abc")
+            .put("source", "test"));
+
+        testPayloadObject
+            .put("messageChild", messageChild)
+            .put("repeatedMessage", new JSONArray()
+                .put(messageChild)
+                .put(messageChild));
+
+        String testPayload = testPayloadObject.toString();
+        byte[] payload = testPayload.getBytes(StandardCharsets.UTF_8);
+        PubsubMessage pm = new PubsubMessage(payload, attributes);
+
+        List<TableRow> attributes = new ArrayList<TableRow>();
+        attributes.add(new TableRow().set("key", "source").set("value", "test"));
+        attributes.add(new TableRow().set("key", "uuid").set("value", "123-456-abc"));
+        attributes.add(new TableRow().set("key", "timestamp").set("value", "2013-08-16T23:36:32.444Z"));
+
+        TableRow childMessage = new TableRow()
+                .set("StringField", "a string")
+                .set("Int32Field", 32)
+                .set("Int64Field", 64)
```

Comment:
Why not a real int64 like 79239016081? So that we really make sure it's not limited to 32 bits...